pages/common/trans.md

```diff
@@ -0,0 +1,23 @@
+# trans
+
+> **Translate Shell** is a command-line translator.
+
+- Translate a word (language is detected automatically):
+
+`trans "{{word_or_sentence_to_translate}}"`
+
+- Get a brief translation:
+
+`trans --brief "{{word_or_sentence_to_translate}}"`
+
+- Translate a word into french:
+
+`trans :fr {{word}}`
+
+- Translate from a specific language:
```

Comment:
Maybe "Translate a word from German to English" sounds more clear and specific.

And why "handy" ?

Reply:
True, will change.
**handy** because:
1. I expect most users of `tldr` to get the english auto-detection when using `trans`, which would result in a dictionary style explanation and not a translation. Thus making the use of `de:en` necessary (in this case)
2. The word might seem english (even to a lot of germans), but in fact is not and so people might learn something 😄

Reply:
Can we use a German-sounding word, so that its clear that we are translating from German to English ?

Reply:
Of course. Do you have a suggestion? As someone who speaks german day-in day-out I might not be the best person to judge the german-ity of german words.
Some suggestions:
- Bier
- Schmetterling
- Kreuzung
- Bahnhof
- Krankenwagen

Reply:
I could guess the first one :wink: The 2nd one looks very german-sounding to me. Another reason to vote for 2nd is because it means butterfly, which is a nice thing.

Reply:
I will take this then. Only _downside_ to this is that **Schmetterling** does not really need the `source` language defined, because google would auto-determine the source language correctly.
But I think that **this** does not matter as much in this case of `tldr`.
Easier to understand is better in this cace.